resources/cdk/eventbridge_rule_with_sns_fanout/README.md

```diff
@@ -0,0 +1,82 @@
+![Stability: Stable](https://img.shields.io/badge/stability-Stable-success.svg?style=for-the-badge)
+
+# Weathertop Producer Stack
+
+This project will create a stack for producing events from EventBrudge and publishing them to a centralized SNS topic.
+
+---
+
+Requirements:
+* git
+* npm (node.js)
+* python 3.x
+* AWS access key & secret for AWS user with permissions to create resources listed above
+  
+---
+
+First, you will need to install the AWS CDK:
+
+```
+$ sudo npm install -g aws-cdk
```

Comment:
Probably want to remove the $ signs from these instructions to make copy/paste easier.